tox.ini

```diff
@@ -82,7 +82,7 @@ commands =
 [testenv:py27-mysql]
 basepython = python2.7
 setenv =
-    SUPERSET__SQLALCHEMY_DATABASE_URI = mysql://root@localhost/superset?charset=utf8
+    SUPERSET__SQLALCHEMY_DATABASE_URI = mysql://mysqluser:mysqluserpassword@localhost/superset
```

Comment:
I believe you'll still need the `?charset=utf8` suffix as Python 2.7 uses ASCII by default.